README.md

```diff
@@ -19,13 +19,20 @@ from thermal_base import utils
 image = ThermalImage(image_path="path/to/image", camera_manufacturer="dji/flir")
```

Comment:
Can you split it into two examples? 
1. One which just shows the thermal image decoding, and then maybe a couple of key variables (ex: the temperature matrix, and the sensor values)
2. One which shows utils from ThermalImageHelpers ( `from thermal_base import utils`) being used